[2024-05-31T16:17:40.530+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-31T16:17:40.553+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_weather_workflow_DAG_v04.transform_data_city_api manual__2024-05-31T11:14:41.858816+00:00 [queued]>
[2024-05-31T16:17:40.564+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_weather_workflow_DAG_v04.transform_data_city_api manual__2024-05-31T11:14:41.858816+00:00 [queued]>
[2024-05-31T16:17:40.565+0000] {taskinstance.py:2306} INFO - Starting attempt 7 of 12
[2024-05-31T16:17:40.580+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform_data_city_api> on 2024-05-31 11:14:41.858816+00:00
[2024-05-31T16:17:40.588+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'my_weather_workflow_DAG_v04', 'transform_data_city_api', 'manual__2024-05-31T11:14:41.858816+00:00', '--job-id', '388', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmp3xvyo7mf']
[2024-05-31T16:17:40.591+0000] {standard_task_runner.py:91} INFO - Job 388: Subtask transform_data_city_api
[2024-05-31T16:17:40.591+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1179) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-31T16:17:40.592+0000] {standard_task_runner.py:63} INFO - Started process 1181 to run task
[2024-05-31T16:17:40.654+0000] {task_command.py:426} INFO - Running <TaskInstance: my_weather_workflow_DAG_v04.transform_data_city_api manual__2024-05-31T11:14:41.858816+00:00 [running]> on host cd9caf1c47f0
[2024-05-31T16:17:40.759+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Khang' AIRFLOW_CTX_DAG_ID='my_weather_workflow_DAG_v04' AIRFLOW_CTX_TASK_ID='transform_data_city_api' AIRFLOW_CTX_EXECUTION_DATE='2024-05-31T11:14:41.858816+00:00' AIRFLOW_CTX_TRY_NUMBER='7' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-31T11:14:41.858816+00:00'
[2024-05-31T16:17:40.760+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-31T16:17:40.798+0000] {python.py:237} INFO - Done. Returned value was: {'file_name': 'city_data_31-05-2024.csv', 'wiki_id': 'Q25315', 'latitude': 10.937, 'longitude': 108.15778}
[2024-05-31T16:17:40.799+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-31T16:17:40.831+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=my_weather_workflow_DAG_v04, task_id=transform_data_city_api, run_id=manual__2024-05-31T11:14:41.858816+00:00, execution_date=20240531T111441, start_date=20240531T161740, end_date=20240531T161740
[2024-05-31T16:17:40.888+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-31T16:17:40.923+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-31T16:17:40.926+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
